lib/charms/layer/jenkins/plugins.py

```diff
@@ -92,6 +92,16 @@ def _install_plugins(self, plugins):
         host.chownr(paths.PLUGINS, owner="jenkins", group="jenkins", chowntopdir=True)
         return plugin_paths
 
+    def _install_hpi_or_jpi(self, plugin, plugin_url_no_suffix):
```

Comment:
Maybe we should keep it as *download* instead of *install*:
`def _download_hpi_or_jpi(self, plugin, plugin_url_no_suffix):`